apps/src/lib/kits/maker/CircuitPlaygroundBoard.js

```diff
@@ -161,16 +161,16 @@ export default class CircuitPlaygroundBoard extends EventEmitter {
         'Cannot initialize components: Not connected to board firmware.'
       );
     }
-
-    return createCircuitPlaygroundComponents(this.fiveBoard_).then(
-      components => {
-        this.prewiredComponents_ = {
-          board: this.fiveBoard_,
-          ...components,
-          ...J5_CONSTANTS
-        };
-      }
-    );
+    return createCircuitPlaygroundComponents(
+      this.fiveBoard_,
+      this.boardType_ === BOARD_TYPE.EXPRESS
```

Comment:
Nit: I wonder if we should pass in the board type here, instead of a boolean? In the (unlikely) event a future board also needs board-specific tweaks.